source/features/branch-buttons.css

```diff
@@ -0,0 +1,11 @@
+.rgh-branch-buttons {
+	margin-left: 0.5rem;
```

Comment:
We should use the same units in both. Let’s go with pixels since the rest of the site does

Reply:
https://github.com/sindresorhus/refined-github/pull/2315/commits/b7496a6ed4673fddf31a07ad67a41c12958b804f should fix most of the styling inconsistencies.